README.md

```diff
@@ -16,48 +16,51 @@ theme/
   ...remaining theme files
 ```
 
-## Project Setup
-
+## Prereqs
 1. Make sure you have [Shopify's themekit](https://shopify.github.io/themekit/) installed globally, `npm i -g themekit`.
 2. Make sure you have [Prepros](https://prepros.io/) version 7 or later installed. 
-3. Put your theme files into the `theme` directory
-4. Get Shopify API Access via a priate app. Instructions can be found under the "Get API Access" setion on [this page](https://shopify.github.io/themekit/). Then copy the password to your private app.
-5. Put your Shopify _store_ and private app _password_ in the `theme/config.yml` file.
-6. Locate your _theme_id_:
-   1. Go to the `Themes` setion of your Shopify store.
-   2. Click on the `Actions` menu next to the theme you are developing and click on `Edit code`.
-   3. You'll now see your theme_id (series of numbers) at the end of the URL. Copy it and paste it your theme/config.yml` file).
-7. If you're not using the current published theme you'll need this extra step to get a proxy URL for Prepros that includes your theme id:
+
+## Project Setup
+
+1. In your shopify store, create two duplicates of your production theme. One will be used while actively developing the theme, while the other will be used to snapshot features/fixes that are ready for the client to preview. Identify the duplicates with a name prefix like  `Seamonster Dev ::`  and `SeaMonster Stage ::`.
+2. Edit `theme/config.yml`. This contains sections that should correspond to the themes that were previously created. 
+   * The `store` follows the pattern `your-store-name.myshopify.com`.
+   * The `password` is a private app password that can be created using the [Themekit Access](https://apps.shopify.com/theme-kit-access?shpxid=a8e67f54-3B47-4818-715D-2431CE697661) app.
+   * Locate your `theme_id`:
+      1. Go to the `Themes` setion of your Shopify store.
+      2. Click on the `Actions` menu next to the theme you are developing and click on `Edit code`.
+      3. You'll now see your theme_id (series of numbers) at the end of the URL. Copy it and paste it your theme/config.yml` file).
+3. Run `yarn prod:pull`. If everything was configured correctly this will pull the production theme files into this project's `theme` folder.
```

Comment:
lets replace all instances of `prod` with `main` for consistency across new projects

Reply:
I'm not 100% on this, just think it might make more sense to use `main` everywhere since that's our prod/master branch

Reply:
How do you feel about renaming the master branch to prod instead. I'm all for the name matching it's intended purpose.

Reply:
I think we've had this conversation multiple times w/in diff. projects. My opinion is that we should stay consistent w/what we've started to establish in other projects which are using `main` as the root branch.